contrib/platform/test/com/sun/jna/platform/win32/Kernel32Test.java

```diff
@@ -539,21 +539,6 @@ public void testCreateProcess() {
         assertTrue(processInformation.dwProcessId.longValue() > 0);
     }
 
-    public void testGetEnvironmentVariable() {
```

Comment:
_Kernel32_ is **very big** - to have a single class test **all** of the API seems too much - that is why in my past submissions (as in this one) I try to group tests related to a specific topic into a separate class (see _NamedPipeTest_) - as its the case for this submission - see _EnvironmentVarsTest._. Regarding your remark, please note that I **did not remove it** (I would never remove a test unless some other test covers it) - I **moved** it to the _EnvironmentVarsTest_ which groups the test for "excercising" the environment variables API(s)